.gitignore

```diff
@@ -35,3 +35,5 @@ Servo.app
 .vscode
 
 /unminified-js
+
+tests/wpt/mozilla/tests/mozilla/microdata/local_tests.sh
```

Comment:
I assume this is a script you wrote for your own testing purposes? Let's not add it to .gitignore; you can just leave it as an untracked file.